schemas/session-replay/browser/stylesheet-schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "session-replay/browser/stylesheet-schema.json",
+  "title": "StyleSheet",
+  "type": "object",
+  "description": "Browser-specific. Schema of a StyleSheet",
+  "required": ["rules"],
+  "properties": {
+    "rules": {
```

Comment:
MDN mentions that `rules` is a legacy property, replaced by `cssRules`: https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet#legacy_properties
Maybe we want to match this?